cms/templates/js/license-selector.underscore

```diff
@@ -0,0 +1,113 @@
+                <% if (optionInfo.type == "boolean") { %>
+                    <% var optionSelected = model.options[optionKey]; %>
+                    <% var optionDisabled = optionInfo.disabled ||
+                            (optionInfo.conflictsWith && _.any(optionInfo.conflictsWith, function(key) {return model.options[key];}));
+                    %>
+                    <li data-option="<%- optionKey %>"
+                        class="action-item license-button
+                               <% if (optionSelected) { print("is-selected"); } %>
+                               <% if (optionDisabled) { print("is-disabled"); } else { print("is-clickable"); } %>"
+                    >
+                        <input type="checkbox"
+                               id="<%- model.type %>-<%- optionKey %>"
+                               name="<%- model.type %>-<%- optionKey %>"
+                               aria-describedby="<%- optionKey %>-explanation"
+                               <% if(optionSelected) { print('checked="checked"'); } %>
+                               <% if(optionDisabled) { print('disabled="disabled"'); } %>
+                        />
+                        <label for="<%- model.type %>-<%- optionKey %>" class="option-name">
+                            <%- optionInfo.name %>
+                        </label>
+                        <div id="<%- optionKey %>-explanation" class="explanation">
+                            <%- optionInfo.help %>
+                        </div>
+                    </li>
+                <% } // could implement other types here %>
+            <% }) %>
+        </ul>
+    </div>
+<% } %>
+
+<% if (showPreview) { %>
+    <div class="license-preview-wrapper">
+        <h4 class="label setting-label">
+            <%= gettext("License Display") %>
+        </h4>
+        <p class="tip">
+            <%= gettext("The following message will be displayed at the bottom of the courseware pages within your course.") %>
+        </p>
+        <div class="license-preview">
+        <% // keep this synchronized with the contents of common/templates/license.html %>
+        <% if (model.type === "all-rights-reserved") { %>
+            © <span class="license-text"><%= gettext("All Rights Reserved") %></span>
+        <% } else if (model.type === "creative-commons") {
+             var possible = ["by", "nc", "nd", "sa"];
+             var enabled = _.filter(possible, function(option) {
+                 return model.options[option] === true || model.options[option.toUpperCase()] === true;
+             });
+             var version = model.options.ver || "4.0";
+             if (_.isEmpty(enabled)) {
+                    enabled = ["zero"];
+                    version = model.options.ver || "1.0";
+             }
+        %>
+            <a rel="license" href="//creativecommons.org/licenses/<%- enabled.join("-") %>/<%- version %>/">
```

Comment:
@singingwolfboy is there a reason why you are using a protocol relative URL here?  There is an advantage to doing that for linking to external resources that have to be fetched, but I don't see the value for a hyperlink.  I was not able to tab to this link from Firefox on Mac (worked fine on Windows).  I would try specifying a protocol either http or https